variables.tf

```diff
@@ -296,6 +296,17 @@ variable "disable_password_authentication" {
   description = "(Optional) Should Password Authentication be disabled on this Virtual Machine? Defaults to `true`. Changing this forces a new resource to be created."
 }
 
+variable "disk_controller_type" {
+  type        = string
+  default     = "SCSI"
```

Comment:
According to the [schema](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/compute/windows_virtual_machine_resource.go#L188-L196), `disk_controller_type` has no default value, why we set it to `SCSI` here?